src/components/basic/ImageGallery/types.ts

```diff
@@ -36,6 +38,5 @@ export interface ImageType {
   modalWidth?: string
   width?: string
   height?: string
-  // eslint-disable-next-line
-  additionalStyles?: any
+  additionalStyles?: CSSProperties
```

Comment:
same as above

Reply:
This change does not affect the portal app’s functionality